api/v1alpha1/groupversion_info.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // Package v1alpha1 contains API Schema definitions for the kubeadm v1alpha1 API group
 // +kubebuilder:object:generate=true
-// +groupName=kubeadm.bootstrap.cluster.sigs.k8s.io
+// +groupName=kubeadm.bootstrap.cluster.x-k8s.io
```

Comment:
Pretty sure this needs to be just `bootstrap.clsuter.x-k8s.io` to work with the default CAPI kubebuilder RBAC settings. Right @vincepri ?

Reply:
that is domain, this is group name. Also unless this is a bug in kubebuilder this has never been touched since generation

Reply:
They need to match

Reply:
See https://github.com/kubernetes-sigs/cluster-api/blob/bf8541c6acdf96bdbd55ce551424923cd1268783/pkg/controller/controller.go#L27

Reply:
Yeah that's correct